ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Idpbuilder based on public information available in blog posts and videos.
```

Comment:
```suggestion
This document lists the organizations that use Idpbuilder
```
We want to encourage any org that uses idpbuilder tool to submit a PR. They might no have public info mentioning. But is encourage. 
Here is an example on ArgoCD list adopters/users https://github.com/argoproj/argo-cd/blob/master/USERS.md